console/src/components/CompactAppList/CompactAppList.js

```diff
@@ -0,0 +1,35 @@
+import kind from '@enact/core/kind';
+import {Cell, Row} from '@enact/ui/Layout';
+import React from 'react';
+import AppIconCell from '../AppIconCell';
+
+const CompactHvac = kind({
+	name: 'CompactHVAC',
+
+	styles: {
+		className: 'compact'
+	},
+
+	handlers: {
+		onTabChange: (ev, {onSelect}) => {
```

Comment:
Does this still function the same way?

Reply:
yea should be

Reply:
Please include `onSelect` in the propTypes definition for this component.